modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Testflow/TaskHeaderActions.tsx

```diff
@@ -60,8 +72,10 @@ const TaskHeaderActions = () => {
 	};
 
 	useEffect(() => {
-		setUsers(taskUser);
-	}, [setUsers, taskUser]);
+		if (testrayTaskUser) {
+			setUsers(testrayTaskUser.map(({user}) => user?.id as number));
```

Comment:
```suggestion
			setUsers(testrayTaskUser.map(({user}) => user?.id as number));
```

Wouldn't it be better "setUsersId¨

Reply:
Agree